tests/test_ocs_basic_install.py

```diff
@@ -181,3 +275,84 @@ def load_config_data(data_path):
     """
     with open(data_path, "r") as data_descriptor:
         return yaml.load(data_descriptor, Loader=yaml.FullLoader)
+
+
+def create_rook_resource(template_name, rook_data, cluster_path):
+    """
+    Create a rook resource after rendering the specified template with
+    the rook data from cluster_conf.
+
+    Args:
+        template_name: name of the ocs-deployment config template.
```

Comment:
add types for params

Reply:
done in https://github.com/red-hat-storage/ocs-ci/pull/23/commits/6659aa4f0c41c1972ffa9f7306ae8b7a8f4f52c1